libbeat/mapping/field_test.go

```diff
@@ -536,7 +536,7 @@ func TestFieldConcat(t *testing.T) {
 				{Name: "foo", Fields: Fields{{Name: "b", Fields: Fields{{Name: "c"}}}}},
 			},
 
-			err: "2 errors: fields contain key <a>; fields contain non object node conflicting with key <foo.b.c>",
+			err: "fields contain key <a>\nfields contain non object node conflicting with key <foo.b.c>",
```

Comment:
I was wondering about this. How does this impact on error rendering to users? I suspect it would be and improvement, but it's worth checking.